x-pack/plugin/build.gradle

```diff
@@ -125,8 +105,29 @@ task copyKeyCerts(type: Copy) {
 sourceSets.test.resources.srcDir(keystoreDir)
 processTestResources.dependsOn(copyKeyCerts)
 
-integTestCluster {
+integTest.runner {
+  /*
+   * We have to disable setting the number of available processors as tests in the same JVM randomize processors and will step on each
+   * other if we allow them to set the number of available processors as it's set-once in Netty.
+   */
+  systemProperty 'es.set.netty.runtime.available.processors', 'false'
+
+
+  // TODO: fix this rest test to not depend on a hardcoded port!
+  def blacklist = ['getting_started/10_monitor_cluster_health/*']
+  boolean snapshot = "true".equals(System.getProperty("build.snapshot", "true"))
```

Comment:
This just looks odd to me. Let's just use `Boolean.valueOf()` as we do in other places.